src/index.js

```diff
@@ -14,7 +14,7 @@ ReactDOM.render(
   </Provider>,
   document.getElementById('root')
 )
-
+require('dotenv').config()
```

Comment:
Is this needed? Usually this is called at the top of a file, where env variables are used. I think we can just use REACT_APP to get environmental variables for free on frontend ... no need for dotenv package. Thoughts?

Reply:
Yeah, I removed it.